### DISCUSSION TITLE
Can't connect QFieldCloud to PostGIS

### DISCUSSION THREAD

**Discussion by sigsdis30** [opengisch/QField#5297] [Q&A]
Hello,

We have installed QFieldCloud v0.26.5 on our own server. The synchronization of projects with geopackages works well. However, we would like to create a project that uses layers from a PostGIS database to update the layers directly in the database.

We noticed that QFieldCloud uses secrets for connecting to a PostGIS database. However, we don't see an option to configure these secrets in the admin interface.

We read in this discussion (https://github.com/opengisch/QField/discussions/4771) that the secrets are not yet accessible from the interface. To connect QFieldCloud to PostGIS, should we revert to an older version? If so, which version would you recommend and is it available? When do you think this feature will be available in the interface? Can the QFieldCloud SDK help to work around this issue?

Thank you for your assistance.

**Comment by boardend** [comment]
Note that secrets can now be added/modified through the admin panel with QFieldCloud version >= 0.27.0

**Reply by rom2049** [reply]
Thank @boardend for your response

There is indeed the possibility to add secrets in the interface with v0.27.0. However, the update in PostGIS is not occurring, and I notice that the 'Value' field of the secrets always remains empty after creation or modification.

**Reply by boardend** [reply]
Let's continue this discussion in https://github.com/opengisch/qfieldcloud/issues/957
